src/modules/objectsearch/ObjectSearchComponent.js

```diff
@@ -19,13 +20,22 @@ export function renderParam(id, props, style) {
         style={{...style}}
         type="text"
         placeholder={I18n.t(`musit.objectsearch.${id}.placeHolder`)}
-        value={props.params[id] || ''}
+        value={props.store.params[id] || ''}
         onChange={(e) => props.onChangeField(id, e.target.value)}
       />
     </FormGroup>
   );
 }
 
+export function callSearchForObjects(props, page) {
```

Comment:
rename to searchForObjects. No need to imply that we are calling the method by adding 'call' to its name.